infrastructure/kube/templates/keep-ecdsa/initcontainer/provision-keep-ecdsa/provision-keep-ecdsa.js

```diff
@@ -220,7 +220,11 @@ async function createSortitionPool(applicationAddress) {
     return await bondedECDSAKeepFactory.methods.getSortitionPool(applicationAddress).call()
   }
 
-  sortitionPoolContractAddress = await bondedECDSAKeepFactory.methods.getSortitionPool(applicationAddress).call()
+  try {
+    sortitionPoolContractAddress = await bondedECDSAKeepFactory.methods.getSortitionPool(applicationAddress).call()
+  } catch (err) {
+    console.error("failed to get sortition pool", err)
```

Comment:
Small note: We don’t want this to continue. Sortition pool creation is the responsibility of the tbtc deploy now, not the initcontainer (this is very recent).